components/unzipped/dashboard/Kanban/KanbanBoard.js

```diff
@@ -0,0 +1,327 @@
+import React, { useState, useEffect, useRef } from "react";
+import styled, { createGlobalStyle } from 'styled-components';
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import { v4 as uuid } from 'uuid';
+import KanbanCard from './KanbanCard';
+import { makeStyles } from "@material-ui/core/styles";
+
+const datas = [
+  { id: 2, label: "aaa" },
+  { id: 3, label: "bbb" },
+  { id: 4, label: "ccc" }
+];
+
+const useStyles = makeStyles({
+  root: {
+    height: 216,
+    flexGrow: 1,
+    maxWidth: 400
+  }
+});
+
+const grid = 8;
+
+const getItemStyle = (isDragging, draggableStyle) => ({
+  // some basic styles to make the items look a bit nicer
+  userSelect: "none",
+  padding: grid * 2,
+  margin: `0 0 ${grid}px 0`,
+
+  // change background colour if dragging
+  background: isDragging ? "lightgreen" : "grey",
+
+  // styles we need to apply on draggables
+  ...draggableStyle
+});
+const getListStyle = (isDraggingOver) => ({
+  background: isDraggingOver ? "lightblue" : "lightgrey",
+  padding: grid,
+  width: 250
+});
+
+// Global Style
+const GlobalStyle = createGlobalStyle`
+  body {
+    font-family: 'Arial', sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+`;
+
+// Styled Components
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+`;
+
+const Board = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  background-color: #ffffff;
+  border-radius: 8px;
+  padding: 0px 20px;
+  width: 100%;
+  height: calc(100vh - 244px); 
+  overflow: scroll;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+
+  ::-webkit-scrollbar {
+    width: 5px;
+  }
+
+  ::-webkit-scrollbar-track {
+    background: #f1f1f1;
+
+  ::-webkit-scrollbar-thumb {
+    background: #888;
+    border-radius: 5px; 
+  }
+
+  ::-webkit-scrollbar-thumb:hover {
+    background: #555;
+  }
+`;
+
+const ColumnContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 10px;
+  width: 300px;
+`;
+
+const ColumnTitle = styled.h2`
+  font-size: 24px;
+  color: #000000;
+  margin: 15px 0px;
+  padding: 0px;
+  text-align: center;
+`;
+
+const DroppableArea = styled.div`
+  background-color: ${(props) => (props.isDraggingOver ? '#34D9B3' : '#e0e4e7')};
+  padding: 8px;
+  width: 100%;
+  min-height: ${(props) => props.minHeight}px; /* Dynamic height */
+  border-radius: 8px;
+`;
+
+// Data and Functions
+const itemsFromBackend = [
+  {
+    id: uuid(),
+    ticketNumber: 101,
```

Comment:
Tickets should be pulled from tasks that are associated with the selected project and department, and should be displayed in the column that corresponds with their selected tag.

Reply:
@Alien4Hire 
Initially, there was a hard-codeded data there. I have continued  with that approach. Currently we're only working with the ui component to make it functional and working features of drag and drop. I'll have these feature get implemented in the current pull request or in the new ticket ?

Reply:
I'll go ahead and merge this, then we can do it in a new pr